### ISSUE TITLE
Gravatars

### ISSUE THREAD

**Issue by geoffreymcgill** [retypeapp/retype#141]
### Discussed in https://github.com/retypeapp/retype/discussions/140

<div type='discussions-op-text'>

<sup>Originally posted by **VBDojnaz** September  9, 2021</sup>
Hey!

It would be great if we could enable using Gravatars as the avatar by default, using the email provided in the author
```
---
author:
  - name: Victor Bäck
    email: vb@manhunt.gg
  - name: Drekec
    email: drekec@manhunt.gg
---
```

With this, it would also be great to be able to use the default avatar option that Gravatar allows with the `d` parameter when an email doesn't have an avatar registered. Example [with valid email](https://www.gravatar.com/avatar/5663eaccee145f9edd23a3601a1c6e3c?d=https%3A%2F%2Fstatic-cdn.manhunt.gg%2Fmanhunt.png), and [without](https://www.gravatar.com/avatar/f3102edf6fa5017b29f25c5953c6e059?d=https%3A%2F%2Fstatic-cdn.manhunt.gg%2Fmanhunt.png). </div>

**Comment by geoffreymcgill** [comment]
Hi @VBDojnaz. Thanks for the feature suggestion!

Using Gravatar as the avatar when an `email` is set and no `avatar` is how we originally envisioned the `author` feature to work, but we just ran out of time within the release schedule to fully implement that particular functionality. 

We will do our best to get this working as envisioned in Retype `v1.10.0`.

**Comment by geoffreymcgill** [comment]
Gravatar support is now available in Retype `v1.10.0` and works as originally requested by @VBDojnaz.

https://retype.com/configuration/page/#author
https://retype.com/configuration/project/#gravatar
